weaviate/connect/v4.py

```diff
@@ -336,10 +371,6 @@ def _get_request_header(self) -> dict:
         """
         return self._headers
```

Comment:
delete this one, too?

Reply:
I left it as an abstraction in case we need to add more but it doesn't do much and if we do it's not an issue